src/tools/ci/install_meson_and_rebase.sh

```diff
@@ -0,0 +1,86 @@
+#! /bin/sh
+
+set -e
+set -x
+
+MESON_REPO=$1
+MESON_BRANCH=$2
+CURRENT_OS=$3
+
+if [ $# -ne 3 ]; then
+    echo "install_meson_and_rebase.sh <MESON_REPO> <MESON_BRANCH> <OS>"
+    exit 1
+fi
+
+case $CURRENT_OS in
+    freebsd|linux|macos|windows|mingw)
```

Comment:
It'd be good to handle net/openbsd here as well.

Reply:
Should I target meson branch instead of meson-upstream-master? There are no net/openbsd tasks on meson-upstream-master.

Reply:
I am planning to work on other comments after your answer.

Reply:
> Should I target meson branch instead of meson-upstream-master? There are no net/openbsd tasks on meson-upstream-master.

Hm. I think we should start to upstream the net/openbsd tasks separately. 

It's probably not worth using my meson branch as base, it ends up having conflicts way more often...